### PR TITLE
Budget App 

### PR DESCRIPTION
**Pull request by zakiyousufi**
### In this project I have created my budget app following these requirements 

 - Requirements [link](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md)
 - Docmantation: [Video Docmantation](https://www.loom.com/share/a3d93229bcc2492788f315f2ce6ee755)

The project deploy running without tailwindcss, If posable I will fix that during weekends

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,214 @@
...
+### Key Features <a name="key-features"></a>
+
+- **[Adding new categories and multiple transactions to each category]**
+- **[Ability to implement crud operations on categories and transactions]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
```
Comment (danielufeli) [inline README.md:91]: - [ ] Great work your put into your video presentation, has a requirement your video should contain a highlight about a piece of the code the you find interesting and are proud of making, also the video length should me max 5mins
- [ ] You live demo link throws error 500 when you click signup, kindly fix.

File: spec/requests/entities_spec.rb [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  end
+
+  before do
+    # stub the authenticate_user! method
+    allow_any_instance_of(EntitiesController).to receive(:authenticate_user!).and_return(true)
+    allow_any_instance_of(EntitiesController).to receive(:current_user).and_return(@user)
+  end
+
+  describe 'GET /index' do
+    before(:example) { get group_entities_path(@group) }
```
Comment (danielufeli) [inline spec/requests/entities_spec.rb:20]: - [ ] Great work you put in writing your tests, kindly check I get the error in the screenshot below when I run test.

test-error

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @zakiyousufi ,

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights
- Nice commits messages✔️
- Clean Design✔️

## Suggested changes
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by zakiyousufi** [comment]
Hi @danielufeli  
Thanks for your kind review and feedback I will fix the marked points

**Pull request merged**